### DISCUSSION TITLE
Allow only logged users...

### DISCUSSION THREAD

**Discussion by martinsnajdr** [Xammie/mailbook#50] [Q&A] [answered]
Hello,

I was trying to allow only logged in users to mailbook, but it seems the user sessions are not loaded for the mailbook routes.

Any idea?

Thank you.

**Comment by Xammie** [comment, marked as answer]
Hello! You can do this with the following config:

```php
'middlewares' => [
    'web',
    'auth',
    \Xammie\Mailbook\Http\Middlewares\RollbackDatabase::class,
],
```

**Reply by martinsnajdr** [reply]
Thank you, I was missing the 'web'.
